assets/model_monitoring/components/src/data_drift_compute_metrics/numerical_data_drift_metrics.py

```diff
@@ -139,71 +139,29 @@ def _normalized_wasserstein(
     production_df_count: int,
     numerical_columns: list,
 ):
-    """Calculate normalized Wassserstein distance for numerical columns."""
-    bin_edges = get_dual_histogram_bin_edges(
-        baseline_df,
-        production_df,
-        baseline_df_count,
-        production_df_count,
-        numerical_columns,
-    )
-    baseline_histograms = get_histograms(baseline_df, bin_edges, numerical_columns)
-    prod_histograms = get_histograms(production_df, bin_edges, numerical_columns)
-
-    baseline_freq_count_map = _calculate_column_value_frequency_bin_map_in_df(
-        baseline_histograms, numerical_columns, baseline_df_count
-    )
-    production_freq_count_map = _calculate_column_value_frequency_bin_map_in_df(
-        prod_histograms, numerical_columns, production_df_count
-    )
-
-    reference_distribution = list(
-        baseline_freq_count_map["feature_bucket_frequency_map"].values()
-    )
-    transformed_production_df = _map_production_df_columns_with_bins(
-        production_df, production_freq_count_map
-    )
-
-    distribution_balance_measures = (
-        DistributionBalanceMeasure()
-        .setSensitiveCols(numerical_columns)
-        .setReferenceDistribution(reference_distribution)
-        .transform(transformed_production_df.select(numerical_columns))
-    )
-
-    output_df = distribution_balance_measures.select(
-        "FeatureName", "DistributionBalanceMeasure.wasserstein_dist"
-    )
-
-    output_df = (
-        output_df.withColumnRenamed("FeatureName", "feature_name")
-        .withColumnRenamed("wasserstein_dist", "metric_value")
-        .withColumn("data_type", F.lit("Numerical"))
-        .withColumn("metric_name", F.lit("NormalizedWassersteinDistance"))
-    )
-
-    # Normalize wasserstein distance
-    baseline_df_stats = baseline_df.summary()
-    baseline_df_stdev = (
-        baseline_df_stats.where(F.col("summary") == "stddev").drop("summary").first()
-    )
-    baseline_df_stdev = baseline_df_stdev.asDict()
-
-    def normalize_value(mapping):
-        def f(wasserstein_dist_col, feature_name):
-            std_dev = float(mapping.value.get(feature_name))
-            norm = max(std_dev, 0.001)
-            return wasserstein_dist_col / norm
-
-        return F.udf(f)
-
-    spark = init_spark()
-    mapping = spark.sparkContext.broadcast(baseline_df_stdev)
+    # TODO: Update to leverage DistributionBalanceMeasure again after reference distribution is supported in SynapeML.
+    # Filter the numerical_columns list to keep only the columns present in both DataFrames
+    common_numerical_columns = [
+        col for col in numerical_columns if col in baseline_df.columns and col in production_df.columns
+    ]
+
+    # Compute the Wasserstein distance for each column
+    rows = {}
```

Comment:
I think rows are a list of lists, as in _ks2sample_pandas_impl, so this line should be `rows = []`

Reply:
Fixed.